src/blockchain_db/lmdb/db_lmdb.h

```diff
@@ -32,6 +32,7 @@
 #include "cryptonote_basic/blobdatatype.h" // for type blobdata
 #include "ringct/rctTypes.h"
 #include <boost/thread/tss.hpp>
+#include <boost/atomic/atomic.hpp>
```

Comment:
Just use `std::atomic` for this.

Reply:
Sounds like a plan. This does yield the question of when and where we use boost-specific equivalents over the standard library. There is quite a lot of code using `boost::shared_ptr`, for example, which might just as well use `std::shared_ptr`. Is this only for legacy reasons?

I know that `boost::thread` is an exception, because it has some functionality that `std::thread` does not, which we actively use. Is that the case for other things as well?

Reply:
Note: I had to also change `boost::thread::id` to `std::thread::id` to be able to use it with `std::atomic`.

Reply:
The main reason we used boost::thread, originally, was because std::thread was broken on Windows, back in gcc 4.x days. I'm not sure that reason still applies.

Reply:
Might want to continue using `boost::atomic<boost::thread::id>` then.